ToggleButton/ToggleButton.html

```diff
@@ -1,4 +1,4 @@
 <template aria-pressed="{{checked}}" attach-point="focusNode" on-click="{{toggle}}">
-	<span attach-point="iconNode" class="d-reset d-inline d-icon {{this.checked && this.checkedIconClass ? this.checkedIconClass : (this.iconClass ? this.iconClass : 'd-hidden')}}"
-	></span><span attach-point="labelNode" class="d-reset d-inline" d-hidden="{{!this.checkedLabel && !this.label}}">{{this.checked && this.checkedLabel ? this.checkedLabel : this.label}}</span>
+	<span attach-point="iconNode" class="d-reset d-inline d-icon {{this.checked &amp;&amp; this.checkedIconClass ? this.checkedIconClass : (this.iconClass ? this.iconClass : 'd-hidden')}}"
+	></span><span attach-point="labelNode" class="d-reset d-inline" d-hidden="{{!this.checkedLabel &amp;&amp; !this.label}}">{{this.checked &amp;&amp; this.checkedLabel ? this.checkedLabel : this.label}}</span>
```

Comment:
Oops, nevermind, I didn't notice it has been already discussed (in https://github.com/ibm-js/deliteful/pull/433#commitcomment-9123284 ).